styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: rgb(64, 184, 224)
+}
```

Comment:
- [ ] Dear @mertkantaroglu, Good job setting up the linters, Burt as you can see your Stylint. one of the reasons is you don't have a `;` at the end of your style on line 2; the other is `Unexpected missing end-of-source newline` on your line. Please fix this to have all linter check passe.

Reply:
> * [ ]  Dear @mertkantaroglu, Good job setting up the linters, Burt as you can see your Stylint. one of the reasons is you don't have a `;` at the end of your style on line 2; the other is `Unexpected missing end-of-source newline` on your line 3. Please fix this to have all linter check passe.

Reply:
Thanks for the heads up Tadesse. I fixed it.

Reply:
Tadesse, it still gives an error and I don't understand the reason. Can you help me out here?

Reply:
Tadesse, I managed to update the README file as you requested but still can't solve the problem with stylelint. It still shows me the same problem. Even though I reinstalled and followed every step. Please help me here, otherwise, I can't move forward with other projects as well.